src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -289,7 +289,35 @@ private static bool IsXrefSpec(JObject obj, JsonSchemaMap schemaMap, out SourceI
         private string GetXrefHref(FilePath file, string uid, int uidCount, bool isRootLevel)
         {
             var siteUrl = _documentProvider.GetSiteUrl(file);
-            return !isRootLevel && uidCount > 1 ? UrlUtility.MergeUrl(siteUrl, "", $"#{Regex.Replace(uid, @"\W", "_")}") : siteUrl;
+            return !isRootLevel && uidCount > 1 ? UrlUtility.MergeUrl(siteUrl, "", $"#{GetStandardizedBookmark(uid)}") : siteUrl;
+        }
+
+        private static string GetStandardizedBookmark(string uid)
+        {
+#pragma warning disable CA1304 // Specify CultureInfo
+            uid = uid.ToLower();
```

Comment:
remove "#pragma warning disable CA1304"
```suggestion
            uid = uid.ToLowerInvarient();
```

Reply:
Fixed.